fiboa_cli/datasets/fieldscapes_corsica_2022.py

```diff
@@ -0,0 +1,139 @@
+# TEMPLATE FOR A FIBOA CONVERTER
+#
+# Copy this file and rename it to something sensible.
+# The name of the file will be the name of the converter in the cli.
+# If you name it 'de_abc' you'll be able to run `fiboa convert de_abc` in the cli.
+
+from ..convert_utils import convert as convert_
+
+# File to read the data from
+# Can read any tabular data format that GeoPandas can read through read_file()
+# Supported protcols: HTTP(S), GCS, S3, or the local file system
+
+# Local URI added to the repository for initial conversion, Original Source https://beta.source.coop/esa/fusion-competition/
+URI = "/home/byteboogie/work/labwork_hkerner/fieldscapes/corsica/boundaries_corsica_2022.gpkg"
+
+# Unique identifier for the collection
+ID = "fieldscapes_corsica_2022"
+# Title of the collection
+TITLE = "Field boundaries for Corsica (Fieldscapes)"
+# Description of the collection. Can be multiline and include CommonMark.
+DESCRIPTION = """ The dataset contains field boundaries for the Corsica."""
+# Bounding box of the data in WGS84 coordinates
+BBOX = [8.684215733091785, 41.726866147253986, 9.55961872699482, 42.808624416493835]
+
+# Provider name, can be None if not applicable, must be provided if PROVIDER_URL is provided
+PROVIDER_NAME = "Republique Francaise - Geoservices"
+# URL to the homepage of the data or the provider, can be None if not applicable
+PROVIDER_URL = "https://data.geopf.fr/telechargement/download/RPG/RPG_2-0__SHP_LAMB93_R94_2022-01-01/RPG_2-0__SHP_LAMB93_R94_2022-01-01.7z.001"
```

Comment:
Did you run this? I don't think we have 7zip support yet. Related to #50

Reply:
Yes, it has to be downloaded and dun on the gpkg inside.

Reply:
We have 7z support now, no manuals steps required.

Reply:
No there was a manual steps involved like creating subset of the original dataset, Not all the fieldscapes dataset are covering the entire original data sources. Hence, this step.